kernel/kernel-api/src/main/java/io/delta/kernel/internal/replay/ActionsIterator.java

```diff
@@ -105,7 +122,7 @@ public boolean hasNext() {
 
   /**
    * @return a tuple of (ColumnarBatch, isFromCheckpoint), where ColumnarBatch conforms to the
-   *     instance {@link #readSchema}.
+   *     instance {@link #deltaReadSchema}.
```

Comment:
Can you update these docs to reflect that we return #checkpointReadSchema batches when `isFromCheckpoint=true`?

Reply:
Done